test/assert.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <stdexcept>
+#include <exception>
+
+// Custom assert function that doesn't call abort
+inline void assert(bool assertion)
```

Comment:
should probably give this another name? if anyone includes assert.h from stdlib chaos ensues. also if you wanna use it at some point, you can't